zipline/utils/run_algo.py

```diff
@@ -71,7 +74,9 @@ def _run(handle_data,
          print_algo,
          metrics_set,
          local_namespace,
-         environ):
+         environ,
+         *args,
+         **kwargs):
```

Comment:
This is for new custom command line arguments in the case where the user adds a new type of base class. However, this is pretty unlikely now that I think of it, as I was expecting this kind of thing to be our responsibility. Any extra subclasses added will simply be options in the list, so that doesn't really change anything. I'll change this back to just blotter.